build/falco/tester/Dockerfile

```
@@ -0,0 +1,37 @@
+FROM golang:1.14-alpine as build
+
+ARG TARGETPLATFORM
+
+ENV GO111MODULE=on \
+    CGO_ENABLED=0  \
+    FALCO_VERSION=0.23.0  \
+    BUILD_TYPE=release
+
+
+RUN apk add --no-cache git make
+
+WORKDIR /go/src/github.com/falcosecurity/falco
+
+RUN export GOOS=$(echo ${TARGETPLATFORM} | cut -d / -f1) && \
+    export GOARCH=$(echo ${TARGETPLATFORM} | cut -d / -f2) && \
+    GOARM=$(echo ${TARGETPLATFORM} | cut -d / -f3); export GOARM=${GOARM:1} && \
```

This is a little neater way of doing it, I'll need to refactor everywhere else also at some point
```suggestion
    GOARM=$(echo ${TARGETPLATFORM} | cut -d / -f3 | cut -c2-) && \

```